.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches: [lt-docusarus]
+    paths: [docs/**]
+
```

Comment:
Let's use this workflow for all that is website/docs related.

Could you rename the file to `website.yml` or `docs.yml` and update the worklfow name accordingly. This workflow with do more than simply deploy the website at some point.

You probably thought about this already, just marking it so we don't forget: branches needs to be updated